IsotropicTextureMat.h

```diff
@@ -32,7 +32,7 @@ class IsotropicTextureMat : public LinIsotropic
   //! \brief Constructor.
   //! \param[in] ps If \e true, assume plane stress in 2D
   //! \param[in] ax If \e true, assume 3D axi-symmetric material
-  inline IsotropicTextureMat(bool ps, bool ax);
+  inline IsotropicTextureMat(bool ps, bool ax) : LinIsotropic(ps,ax) { } ;
```

Comment:
Yes, and if you also remove the semi-colon at the end and the space between the braces, I'm happy